mgmt/smp/transport/smp_uart/src/smp_uart.c

```diff
@@ -23,47 +23,53 @@
 #include <bsp/bsp.h>
 
 #include <mgmt/mgmt.h>
-#include <newtmgr/newtmgr.h>
+#include <smp/smp.h>
```

Comment:
if `smp/smp.h` was `mynewt_smp/smp.h` some of the build errors would be gone

Reply:
removed smp_uart